tools/update_cli_readme_documentation.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Tool to automatically update README.md files that contain --help output of c++ TCLAP binaries
+
+import logging
+import argparse
+import os
+import pathlib
+import re
+import sys
+import subprocess
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser(description='Tool to update README.md CLI documentation.')
+  parser.add_argument(
+      '--binary',
+      required=True,
+      help='Relative path to the target binary, for example: "bazel-bin/nighthawk_client".')
+  parser.add_argument(
+      '--readme',
+      required=True,
+      help='Relative path to the target documentation file, for example: "README.md"')
+  parser.add_argument(
+      '--mode',
+      default="check",
+      required=True,
+      choices={"check", "fix"},
+      help='Either "check" or "fix"')
+
+  args = parser.parse_args()
+  root = logging.getLogger()
+  root.setLevel(logging.INFO)
+
+  handler = logging.StreamHandler(sys.stdout)
+  handler.setLevel(logging.INFO)
+  formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
Why do we have this complex logging for a simple script like this?

Reply:
Simplified